src/Dashboard/Types/NodesStatistics/Statistic.php

```diff
@@ -190,8 +201,10 @@ protected function prepareChartData()
      * @param int $length
      * @return string
      */
-    private static function randomString($length = 16) {
+    private static function randomString($length = 16)
```

Comment:
No one added this in this merge btw.
I was there before, I just removed the out-commented line.

I don't see the problem. Fe wanted a random string without numbers, they got it